mix.exs

```diff
@@ -49,6 +49,8 @@ defmodule Orbit.MixProject do
       {:floki, ">= 0.30.0", only: :test},
       # used by ex_aws
       {:hackney, "== 1.20.1"},
+      {:guardian, "== 2.3.2"},
+      {:guardian_phoenix, "== 2.0.1"},
```

Comment:
These aren't alphabetized (should be before `hackney`)

Reply:
fixed in [faa8eea](https://github.com/mbta/orbit/pull/21/commits/faa8eea2414044416f2e2a3086988bc29af454b4)